### DISCUSSION TITLE
How to downgrade nhost project plan Pro to Starter?

### DISCUSSION THREAD

**Discussion by connectedlife-ro** [nhost/nhost#2531] [Q&A]
I want to downgrade Nhost project plan to starter

**Comment by dbarrosop** [comment]
Downgrading isn't supported as we can't guarantee the database will fit. If you know it will you can:

1. create a free project
2. do a pgdump on your old project
3. restore the dump your new project
4. update your application to point to the new subdomain
5. delete old project.

If you have further questions don't hesitate to let us know.
